code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -85,6 +86,12 @@
 		stop_automated_movement = 1
 		walk_to(src,movement_target,0,3)
 
+/mob/living/simple_animal/cat/death()
+	.=..()
+	stat |= DEAD

```

Comment:
I forget whether stat is a bitfield or not, but I don't think I've ever seen bitfield operations done with it. It's always either 0, 1, or 2. Unless you can find me a reference, it would probably be best to use `stat = DEAD`. I know that the majority of checks concerning stat use an equals operator, instead of greater than or equal to.